fuzzbench/run_experiment.py

```diff
@@ -26,8 +26,18 @@ def run_experiment():
     print('Initializing the job queue.')
     queue = rq.Queue()
     jobs = []
-    for i in range(6):
-        jobs.append(queue.enqueue(fake_jobs.build_image, 'something-%d' % i))
+    jobs.append(
+        queue.enqueue(fake_jobs.build_image, 'base-image', job_id='base-image'))
```

Comment:
Should we rename fake_jobs module, why not just jobs module.

Reply:
+1